docs-next/types.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module '@mdx-js/react' {
```

Comment:
can't we just install [mdx-js-react types](https://www.npmjs.com/package/@types/mdx-js__react) instead of declaring manually?

Reply:
good catch, thnx @kschingiz